### DISCUSSION TITLE
how to get cross-section after cuts ?

### DISCUSSION THREAD

**Discussion by Purnath** [MadAnalysis/madanalysis5#275] [Q&A]
Dear ma5 team,
I'm implementing cuts and I'm getting the no. of events.  Instead, can I get a cross-section after the cuts?
Also, when I'm plotting the histograms, they get plotted with the number of events in each bin but I want the distribution to be with differential cross-section. Can this be done directly?
I also want to do this for a particular luminosity.
I tried using the set main luminosity but that part is not clear to me too. Can you explain how the normalisation to luminosity is being done?

**Comment by BFuks** [comment]
Hello,

A simple way is to fix the bin size in histograms and the luminosity so that the content of any bin or the number of events after each cut would be equal to the cross section.

For the rest, I refer to the manual.

Regards,

Benjamin
